### PR TITLE
Adding Sysbench Spec for System Test

### PR DESCRIPTION
**Pull request by jainsmit**
**What this PR does / why we need it**:
1. This PR add new directory for System Test Spec.
2. This PR adds sysbench specs for system test.

**Which issue(s) this PR fixes** (optional)
Closes #PTX-12605

**Special notes for your reviewer**:

### CODE REVIEW

File: drivers/scheduler/k8s/systemtestSpec/sysbench/px-sysbench-app.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+metadata:
+  name: sysbench-mysql-data
+spec:
+  storageClassName: sysbench-sc-secure
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Ti 
+bash-3.2$ cat px-sysbench-app.yml
```
Comment (mborodin-px) [inline drivers/scheduler/k8s/systemtestSpec/sysbench/px-sysbench-app.yml:12]: I guess this is not correct yaml

File: drivers/scheduler/k8s/systemtestSpec/sysbench/px-sysbench-app.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+apiVersion: v1
+metadata:
+  name: sysbench-mysql-data
+spec:
+  storageClassName: sysbench-sc-secure
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Ti
```
Comment (Santhoshairody) [inline drivers/scheduler/k8s/systemtestSpec/sysbench/px-sysbench-app.yml:11]: Can we reduce the size from 1Ti to 512Gi as initial setup won't be having pool size 1Ti+

File: drivers/scheduler/k8s/systemtestSpec/sysbench/px-sysbench-app.yml (hunk 1 above)
Comment (Santhoshairody) [inline drivers/scheduler/k8s/systemtestSpec/sysbench/px-sysbench-app.yml:12]: Is this line valid line

Comment (jainsmit) [reply drivers/scheduler/k8s/systemtestSpec/sysbench/px-sysbench-app.yml]: Fixed this ...Because of copy and paste from shell

Comment (jainsmit) [reply drivers/scheduler/k8s/systemtestSpec/sysbench/px-sysbench-app.yml]: This was wrong yaml.

**Review by Santhoshairody — APPROVED** [review]

**Pull request merged**